subsys/logging/Kconfig.template.log_format_config

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2022 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config LOG_BACKEND_$(backend)_OUTPUT_DICTIONARY
+	bool
+	depends on LOG2
+	select LOG_DICTIONARY_SUPPORT
+	help
+	  Backend is in dictionary-based logging output mode. Supported only for UART backend.
+
+choice "LOG_BACKEND_$(backend)_OUTPUT"
+	prompt "Backend Output Mode"
+	default LOG_BACKEND_$(backend)_OUTPUT_TEXT
+
+config LOG_BACKEND_$(backend)_OUTPUT_TEXT
+	bool "Text"
+	help
+	  Output in text.
+
+config LOG_BACKEND_$(backend)_OUTPUT_SYST
+	bool "MIPI SyS-T"
+	depends on LOG_BACKEND_$(backend)
+	depends on LOG_MIPI_SYST_ENABLE
+	help
+	  When enabled backend is used to output syst format logs.
+
+config LOG_BACKEND_$(backend)_OUTPUT_DICTIONARY_HEX
+	bool "Dictionary (hexadecimal)"
+	depends on LOG2
+	select LOG_BACKEND_$(backend)_OUTPUT_DICTIONARY
+	help
+	  Dictionary-based logging output in hexadecimal. Supported only for UART backend.
```

Comment:
this one is for UART only. Maybe it shouldn't be in template but only added in uart configuration.

Reply:
@nordic-krch Moved this to uart configuration. Thanks.